src/Uno.UWP/Devices/Geolocation/GeolocatorHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if __IOS__
```

Comment:
You can name the file GeolocationHelper.iOS.cs, it'll remove the `#if`.

Reply:
done!